crates/connectors/ndc-postgres/src/configuration_mapping.rs

```diff
@@ -0,0 +1,15 @@
+pub(crate) fn convert_isolation_level(
```

Comment:
Maybe something like this?

```suggestion
//! A module for converting ndc-configuration data type into query-engine data types.

/// Convert a user-specified configuration of the isolation level for transactions
/// into a SQL data type representing that isolation level, which will be passed
/// to the engine on requests.
pub(crate) fn convert_isolation_level(
```

Reply:
I love it, thanks!